### DISCUSSION TITLE
Aries planner alternating between sequential and temporal

### DISCUSSION THREAD

**Discussion by MannavaVivek** [aiplan4eu/unified-planning#601] [Q&A]
I'm currently working on a planning task using the Aries planner to solve a hierarchical planning problem. I've encountered a weird issue where the planner seems to inconsistently determine whether the same problem and domain should be chosen sequentially or temporally.

To be more specific, when the planner identifies the problem as sequential, it generates a proper solution. However, sometimes, it interprets the problem and domain as temporal, leading to the following output with no plan

Plan found:
Hierarchical TimeTriggeredPlan:

And this is how I am choosing the operation mode:

OneshotPlanner(name='aries').solve(pb, output_stream=sys.stdout)  

I have also tried passing the problem kind argument, but that seems to change nothing. So my question is how do I specify that my problem should be solved sequentially?

**Comment by arbimo** [comment]
Aries always interpret the problem as temporal and will post-process the resulting time-triggered plan into a sequential plan when there was no temporal features in the initial problem. 

Could you provide enough information and code to reproduce the problem ? (it may also lie in the conversion from protobuf to python provided by the UP)

**Comment by MannavaVivek** [comment]
I rewrote the tasks and methods assuming I was not setting the preconditions right, and it seemed to fix this issue. So I will close this issue for now, but will update if I face this issue again.
